routes/runtime/github.rb

```diff
@@ -124,7 +124,7 @@ class Clover
           retries = 0
           begin
             upload_id = repository.blob_storage_client.create_multipart_upload(bucket: repository.bucket_name, key: entry.blob_key).upload_id
```

Comment:
test

Reply:
Added a test and rebased.

Reply:
oh, sorry, no, I was testing the merge-with-comments. My bad :)

Reply:
but hey, a test, sure